proxy/gateway.go

```diff
@@ -104,7 +120,9 @@ func (gw *Gateway) buildRouter() {
 }
 
 func (gw *Gateway) Handle(w http.ResponseWriter, r *http.Request) {
-	source, _ := gw.router.Execute(r).(*entities.Source)
+	var source *entities.Source
+
+	source, _ = gw.router.Execute(r).(*entities.Source)
```

Comment:
```suggestion
	source := gw.router.Execute(r).(*entities.Source)
```

gw.router.Execute(r) has only one return value

Reply:
`source, _ = gw.router.Execute(r).(*entities.Source)` should keep for `proxy listen` test expected 404 
`test/proxy/listen_test.go:29`